scripts/support/esp32.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+
+cmd_flash() {
+	local esptool=${ESP_IDF_PATH}/components/esptool_py/esptool/esptool.py
```

Comment:
should allow for esptool path to be set by user if desired, and default to what you have here.